src/assets/json/checklist/tests-ios-en.json

```diff
@@ -0,0 +1,155 @@
+	"resultat": ["Only informative images are presented."],
+	"exception": "",
+	"raccourcis": "",
+	"moreInfo": "",
+	"profils": ["Checklist initiale"],
+	"wcag": ["1.1.1"],
+	"ID": "testMobID-1",
+	"IDorigin": "testMobID-1"
+}, { 
+	"themes": "Colors and contrasts",
+	"title": "Provide enough contrast between text and background",
+	"type": ["Color Contrast Analyser"], 
+	"tests": ["Take a screenshot.", "Inspect the screenshot with Color Contrast Analyser on PC or Mac."], 
+	"verifier": ["Contrast is sufficient to identify texts or images of text."], 
+	"resultat": ["<strong>Presence of text elements or images in text form with sufficient contrast:</strong><ul><li>Size less than <code>18pt</code> (body text): minimum contrast of 4.5:1 (7:1 recommended).</li><li>Size greater than or equal to <code>18pt</code> (headings): minimum contrast of 3:1 (4.5:1 recommended).</li></ul></li><li><strong>For bold text:</strong> <ul><li>Size less than <code>14pt</code>: minimum contrast of 4.5:1 (7:1 recommended).</li><li>Size greater than or equal to <code>14pt</code>: minimum contrast of 3:1 (4.5:1 recommended).</li></ul>"],
+	"exception": "Text that is part of an inactive user interface component or a logo, a brand name.", 
+	"raccourcis": "", 
+	"moreInfo": "https://a11y-guidelines.orange.com/en/mobile/ios/design/#colours", 
+	"profils": ["Checklist initiale"], 
+	"wcag": ["1.4.3"], 
+	"ID": "testMobID-2", 
+	"IDorigin": "testMobID-2" 
+}, { 
+	"themes": "Colors and contrasts",
+	"title": "Make sure the color is not used as the only means of conveying information", 
+	"type": ["Manual test"], 
+	"tests": ["Identify elements using color to convey information."],
+	"verifier": ["Color is not the only means of conveying information: at least one other visual method is available to obtain the same information."], 
+    "resultat": ["Information conveyed by color can also be obtained by means of an explicit text.", "The information conveyed by color is complemented by other visual information (e.g., icons use different colors, but also different shapes).", "Special case of links in text: if they are not underlined, provide a means other than color to distinguish them on keyboard focus and mouse hover."], 
+	"exception": "", 
+	"raccourcis": "", 
+	"moreInfo": "https://a11y-guidelines.orange.com/en/mobile/ios/design/#colours", 
+	"profils": ["Checklist initiale"], 
+	"wcag": ["1.4.1"], 
+	"ID": "testMobID-3", 
+	"IDorigin": "testMobID-3" 
+}, {
+	"themes": "Keyboard navigation",
+	"title": "Allow the use of the application with keyboard",
+	"type": ["Bluetooth keyboard or Switch Control"],
+	"tests": ["Connect a Bluetooth keyboard to the iOS device (or use Switch Control).<br>See <a target=_blank href=https://a11y-guidelines.orange.com/en/articles/how-to-test-ios/>the article on getting started with features on iOS (new tab)</a>.","Navigate the page using the keyboard keys <kbd>tab</kbd> ou <kbd>shift + tab</kbd>."],  

```

Comment:
~~~suggestion
	"tests": ["Connect a Bluetooth keyboard to the iOS device (or use Switch Control).<br>See <a target=_blank href=https://a11y-guidelines.orange.com/en/articles/how-to-test-ios/>the article on getting started with features on iOS (new tab)</a>.","Navigate the page using the keyboard keys <kbd>tab</kbd> ou <kbd>shift</kbd> + <kbd>tab</kbd>."], 
~~~